docs/CODEOWNERS

```diff
@@ -0,0 +1,12 @@
+# This is a comment.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+*       @ottodevs @Quazia @rkzel @topocount @chadoh @Schwartz10
+
+# Order is important; the last matching pattern takes the most
+# precedence. When someone opens a pull request that only
+# modifies solidity files, only the following owners will be
+# requested for a review.
+*.sol    @Quazia @Schwartz10 @topocount @rkzel
```

Comment:
@Quazia does this list look correct to you?

Reply:
I think it should probably be @Quazia @ottodevs @topocount